backtrace-library/src/main/java/backtraceio/library/BacktraceClient.java

```diff
@@ -150,9 +154,15 @@ public void send(BacktraceReport report) {
      */
     public void send(BacktraceReport report, OnServerResponseEventListener
             serverResponseEventListener) {
-        super.send(report, serverResponseEventListener);
+        super.send(report, serverResponseEventListener, this.isProguardEnabled);
     }
 
+    /**
+     * Inform Backtrace API that we are using Proguard symbolication
+     */
+    public void enableProguard() {
```

Comment:
Why we don't want to move `isProguardEnabled ` to `BacktraceBase` class? Advantages that I see:
* Even if someone will try to implement their own version of Backtrace integration by using `BacktraceBase` class, `isProguardEnabled` will still be necessary to add/implement.
* We will avoid passing isProguardEnabled flag every time to `BacktraceBase`
* More code in BacktraceClient class because of support  there for isProguardEnabled flag.

Reply:
That makes sense, I will make this change

Reply:
I see this is much cleaner, thank you for this helpful feedback!